File: Data Structures & Algorithms/majority-element-ii/submission-1.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        numCount = {}
        n = len(nums)
        
        for num in nums:
            numCount[num] = 1 + numCount.get(num, 0)

        ans = []
        for key, value in numCount.items():
            if value > (n / 3):
                ans.append(key)

        return ans

File: Data Structures & Algorithms/reverse-bits/submission-1.cpp
class Solution {
public:
    uint32_t reverseBits(uint32_t n) {
        int ans{};

        for (int i = 0; i < 32; i++) {
            int tmp1 = (1 << i) & n;

            if (tmp1) {
                ans |= (1 << (31 - i));
            }
        }

        return ans;
    }
};
